Assets/Tests/InputSystem/Plugins/XRTests.cs

```diff
@@ -465,7 +471,10 @@ private static InputDeviceDescription CreateSimpleDeviceDescriptionByRole(Device
             manufacturer = "Manufacturer",
             capabilities = new XRDeviceDescriptor
             {
+#if UNITY_2019_3_OR_NEWER
```

Comment:
Why not `#if !UNITY_2019_3_OR_NEWER`?

Reply:
Was going to put somethig in, but it wasn't required for those test cases.
Fixing with that^